src/assets/config_files/usecases.json

```diff
@@ -115,17 +115,51 @@
     ],
     "subcellular": [
       {
-        "title": "KappaNeuron how to",
-        "description": "An example use case on how to use KappaNeuron",
+        "title": "KappaNEURON demonstration",
+        "description": "A simple example use case of how to use KappaNEURON for a calcium pump",
         "experience": ["all"],
         "maturity": ["beta"],
         "disabled": true,
         "picture": {
-          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/placeholder.jpg"
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/KappaNEURON.png"
+          "alt": "Thumbnail of KappaNEURON traces"
         },
         "next": "ta_form",
         "dataprotected": true,
-        "files": []
+        "files": [
+          {
+            "file": "7e4731f8-0e6d-4548-992d-136d9aef7fc4",
```

Comment:
I would suggest ff it's possible I would suggest to put these 3 files in a zip and then fetch only once form the Collab Storage and unzip it on the notebook. That is usually faster than talk with the Collab Storage 3 times.